tweakwcs/correctors.py

```diff
@@ -474,7 +474,8 @@ def tanp_to_det(self, x, y):
             ra = np.atleast_1d(ra)
             dec = np.atleast_1d(dec)
         x, y = self._wcs.all_world2pix(ra, dec, 0, tolerance=1e-6, maxiter=50)
-        if not ndim:
+        if not ndim and np.size(x) == 1:
+            # convert to scalars:
             x = float(x)
             y = float(y)
```

Comment:
```suggestion
            x = float(x[0])
            y = float(y[0])
```
The warnings seen here can be fixed by the above suggestion:
https://dev.azure.com/spacetelescope/tweakwcs/_build/results?buildId=5268&view=logs&j=ac09f1b7-6dd6-5dc4-c15c-72e6db99c59d&t=074acbf2-bfd8-595c-a132-8efb0c395263&l=495
I'm not sure when this will be an error but I wouldn't be surprised if it ends up in numpy 2.0:
```
tweakwcs/tests/test_correctors.py::test_fitswcs_coord_transforms
  /Users/runner/work/1/s/tweakwcs/correctors.py:479: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
    x = float(x)
```